### PR TITLE
fix(DB/quest_poi_points): Fix The Battered Hilt Questline

### PR DESCRIPTION
**Pull request by LukasVolgger**
## CHANGES PROPOSED:
-  Fix the quest poi points for all Horde/Alliance quests as they were completely messed up

**The following quest are hereby fixed:**
- 24555
- 14444
- 24557
- 14457
- 24556
- 20438
- 24800
- 24801
- 24796
- 24461
- 24480

Examples: 
image

2020-12-04 11_04_46-Window

2020-12-04 10_56_32-Window

## ISSUES ADDRESSED:
- None

## TESTS PERFORMED:
- Applied SQL script
- Tested all quests ingame

## HOW TO TEST THE CHANGES:

All ID's from above need to be tested!

1. .q rem & .q a (Pick an quest id from above)
2. Check quest poi point (Open your world map and go to the location)
3. .q c (Quest id which you chose to test)
4. Check quest poi point again (Open your world map and go to the location)

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Review by Helias — APPROVED** [review]

**Comment by Knindzagxg** [comment]
Nice one mate.

**Review by Knindzagxg — APPROVED** [review]

**Comment by gengarshadowball** [comment]
Good job, however on the first quest there's no POI so you have to read the quest log to know you must go to the Wyrmrest Temple.

**These quests could have more accurate POI aswell:**

## Reforging the blade
image

## The Halls of Reflection
image

**Comment by LukasVolgger** [comment]
Thx, i'll take a look at this 2!

**Comment by LukasVolgger** [comment]
Horde site was already fixed that's why i didn't recognize it... because i'm only Horde ^-^

Now these 2 quests are fixed too 👍🏻 
2020-12-06 09_50_11-World of Warcraft

**Comment by LukasVolgger** [comment]
Since all the other quests are good, i think we can merge this 👍🏻

**Comment by gengarshadowball** [comment]
Good job 👍

**Review by Si1ker — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1607085889307075600.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+UPDATE `quest_poi_points` SET `X`=5800, `Y`=792 WHERE `QuestID`=24801 AND `Idx1`=0 AND `Idx2`=0;
+
+-- A Victory For The Silver Covenant - Alliance
+UPDATE `quest_poi_points` SET `X`=5796, `Y`=799 WHERE `QuestID`=24796 AND `Idx1`=0 AND `Idx2`=0;
+
+-- Reforging The Sword - Alliance
+UPDATE `quest_poi_points` SET `X`=5652, `Y`=2106 WHERE  `QuestID`=24461 AND `Idx1`=6 AND `Idx2`=0;
+
+-- The Halls Of Reflection - Alliance
+UPDATE `quest_poi_points` SET `X`=5652, `Y`=2106 WHERE  `QuestID`=24480 AND `Idx1`=2 AND `Idx2`=0;
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1607085889307075600.sql:34]:
```suggestion
UPDATE `quest_poi_points` SET `X`=5652, `Y`=2106 WHERE  `QuestID`=24480 AND `Idx1`=2 AND `Idx2`=0;


```

**Pull request merged**